src/main/java/org/prebid/server/bidder/avocet/AvocetBidder.java

```diff
@@ -42,9 +42,9 @@ public AvocetBidder(String endpointUrl, JacksonMapper mapper) {
 
     @Override
     public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
-        final String body;
+        final byte[] body;
         try {
-            body = mapper.encode(request);
+            body = mapper.encodeToBytes(request);
```

Comment:
1. Pls inline the variable.
2. i'd say it's not required to be safe here, pls remove `try-catch`.